### DISCUSSION TITLE
lefthook unistall does not remove lefthook.checksum

### DISCUSSION THREAD

**Discussion by JuliusHenke** [evilmartians/lefthook#366] [Q&A] [answered]
I found a bug in the `lefthook uninstall` command. It does not remove the `.git/info/lefthook.checksum` file.

This is a problem since following order of commands results in no hooks being installed into `.git/hooks`:
1.  `lefthook install`
2. `lefthook uninstall`
3. `lefthook install`

I would like to try to fix this with a PR, if you are cool with it? 😃

**Comment by mrexox** [comment, marked as answer]
@JuliusHenke yeah! This would be nice! I'll be waiting for a PR :)
